src/app/campaign/availability/availability.component.ts

```diff
@@ -1,59 +1,97 @@
-              (mouseleave)="toggleZoneWeekHover(zone.zone, week.startDate)">
-              <prx-icon
-                [class.hide]="!(zoneWeekHover[zone.zone + '-' + week.startDate] || zoneWeekExpanded[zone.zone + '-' + week.startDate])"
-                name="arrows-alt-v" size="6px" color="primary">
-              </prx-icon>
-              {{ week.startDate | date:'M/dd' }}
-            </button>
-          </div>
-          <div class="avail">{{ (week.allocated + week.availability) | largeNumber }}</div>
-          <div class="goal"></div>
-          <div class="edit">
-            <button class="btn-link" aria-label="Edit">
-              <prx-icon name="pencil" size="14px" color="primary"></prx-icon>
-            </button>
-          </div>
+    <ng-template #inventory>
+      <grove-goal-form [flight]="flight" (goalChange)="goalChange.emit($event)"></grove-goal-form>
+      <p class="error" *ngIf="allocationPreviewError">Got error #{{ allocationPreviewError.status }} from allocation preview</p>
+      <section *ngFor="let zone of availabilityZones">
+        <h3>{{ getZoneName(zone.zone) }}</h3>
+        <div class="row head">
+          <div class="date">Week</div>
+          <div class="avail">Available</div>
+          <div class="goal">Goal</div>
+          <div class="edit"><prx-icon name="lock" size="14px" color="secondary"></prx-icon></div>
         </div>
-        <ng-container *ngIf="zoneWeekExpanded[zone.zone + '-' + week.startDate]">
-          <div class="row expand" *ngFor="let day of week.groups">
-            <div class="date"><span>&mdash;</span><span>{{ day.startDate | date:'M/dd' }}</span></div>
-            <div class="avail"><span>&mdash;</span><span>{{ (day.allocated + day.availability) | largeNumber }}</span></div>
-            <div class="goal"></div>
-            <div class="edit"></div>
+        <ng-container *ngFor="let week of zone.totals.groups">
+          <div class="row" [class.row-highlight]="week.allocated + week.availability < week.allocationPreview">
+            <div class="date">
+              <button
+                class="btn-link"
+                (click)="toggleZoneWeekExpanded(zone.zone, week.startDate)"
+                (mouseover)="toggleZoneWeekHover(zone.zone, week.startDate)"
+                (mouseleave)="toggleZoneWeekHover(zone.zone, week.startDate)"
+              >
+                <prx-icon
+                  [class.hide]="!(zoneWeekHover[zone.zone + '-' + week.startDate] || zoneWeekExpanded[zone.zone + '-' + week.startDate])"
+                  name="arrows-alt-v"
+                  size="6px"
+                  color="primary"
+                >
+                </prx-icon>
+                {{ week.startDate | date: 'M/dd' }}
+              </button>
+            </div>
+            <div class="avail">{{ week.allocated + week.availability | largeNumber }}</div>
+            <div class="goal">
+              {{
```

Comment:
Show the allocation preview if present (even if 0), otherwise the availability allocated should be shown.

The eventual intention here is that the allocation preview will be a temporary state we hold for the flight when looked up (by changing goal, daily min or flight.) Then once allocated upon saving the state, we should clear the allocation preview state for the flight in order to show what has been allocated.